src/hidimstat/ensemble_clustered_inference.py

```diff
@@ -113,11 +112,6 @@ def ensemble_clustered_inference(
         Number of CPUs used to compute several clustered inference
         algorithms at the same time.
 
-    memory : str, optional (default=None)
```

Comment:
Again

Reply:
see above

Reply:
Please keep the memory argument explicit.

Reply:
done